controllers/datadogagent/clusteragent.go

```diff
@@ -528,6 +528,10 @@ func getEnvVarsForClusterAgent(logger logr.Logger, dda *datadoghqv1alpha1.Datado
 			Name:  apicommon.DDLeaderLeaseName,
 			Value: utils.GetDatadogLeaderElectionResourceName(dda),
 		},
+		{
+			Name:  apicommon.DDKubeResourcesNamespace,
+			Value: utils.GetDatadogAgentResourceNamespace(dda),
+		},
```

Comment:
Hi, this file configures the cluster agent for the old controller of the operator that we're trying to move away from in favor of the newer version that was GA'ed in v1.0.0. Could you instead make the change for the newer operator version? Adding the env var under `defaultEnvVars` in [this file](https://github.com/DataDog/datadog-operator/blob/7bd890e16e83176f5cd085e39e624d040172fd45/controllers/datadogagent/component/clusteragent/default.go#L121) instead should make the equivalent change.